serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel.
+Channels are Kubernetes Custom Resources (CRs) which define a single event forwarding and persistence layer.
+After events have been sent to a channel, these events can be sent to multiple Knative services by using a subscription.
+
+The default configuration for channel instances is defined in a ConfigMap by the cluster administrator, however developers can still create their own channels directly by instantiating a supported channel object.
```

Comment:
I guess we should leave the "by the cluster administrator" part out, as we have removed the configmap part from the docs... 

perhaps just "The default configuration for channel instances is defined in the default-ch-webhook ConfigMap, however ..."  with no explicit mention that this should be changed by anyone?  (as it may be still useful to know this, especially if users install the dev-preview Eventing Kafka Operator, which modifies the default-ch-webhook ConfigMap)